frontend/src/views/DecompilationView.svelte

```diff
@@ -27,15 +33,26 @@
         $selectedResource.attributes[
           "ofrak.model._auto_attributes.AttributesType[DecompilationAnalysis]"
         ]["decompilation"],
-        { language: "c" }
+        { language: "c" },
       ).value;
       return html;
     } else {
-      return 'To see decompilation, click "Analyze" on the left toolbar';
+      decompilation = "Decompiling...";
+      return await $selectedResource.analyze().then(() => {
+        var html = hljs.highlight(
+          $selectedResource.attributes[
+            "ofrak.model._auto_attributes.AttributesType[DecompilationAnalysis]"
+          ]["decompilation"],
+          { language: "c" },
+        ).value;
+        return html;
+      });
```

Comment:
```suggestion
     await $selectedResource.analyze();
     return hljs.highlight(
       $selectedResource.attributes[
         "ofrak.model._auto_attributes.AttributesType[DecompilationAnalysis]"
       ]["decompilation"],
       { language: "c" },
     ).value;
```

Since the parent function is already async, we don't need a `.then` with callback.